### PR TITLE
Add profile for amehi0index

### PR DESCRIPTION
**Pull request by amehi0index**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/997"></a>

### CODE REVIEW

File: public/data/amehi0index.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "url": "https://github.com/amehi0index",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/amehi0index",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
```
Comment (Panquesito7) [inline public/data/amehi0index.json:23]: I think it'd look better if you capitalize the `i` here 😅 What do you think? Thanks.
```suggestion
      "name": "Follow me on LinkedIn",
```

File: public/data/amehi0index.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Amelia hill",
```
Comment (Panquesito7) [inline public/data/amehi0index.json:2]: I think it'd be best if you capitalize the `h` here 😅 What do you think? Thanks.
```suggestion
  "name": "Amelia Hill",
```

Comment (amehi0index) [reply public/data/amehi0index.json:2]: I think you are absolutely correct! I was sooo confused trying to figure everything out that I missed those. Thank you!

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by Pradumnasaraf — APPROVED** [review]
LGTM

**Pull request merged**